.babelrc

```diff
@@ -1,5 +1,7 @@
 {
   "presets": [
-    "es2015"
+    [
+      "es2015",
```

Comment:
If this equals to "presets": ["es2015"] then it no need for pull request.
Chenges should be minimal.